autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
> Adding :verbose in front of the command is basically saying "tell me everything" in most of Vim's builtin commands.

Not really.  `:verbose` takes a count which acts like setting `&verbose` to that number.  That is, `:10verbose {cmd}` is like `:set verbose=10 | {cmd} | set verbose=0`.  It does make sense that `:verbose Neomake` should have more logging than typical, but that shouldn't preclude supporting multiple levels of verbosity.